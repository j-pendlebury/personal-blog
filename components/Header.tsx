const Header = () => {
  return (
    <div className="header">
      <div className="gradient-border-line"></div>
      <nav>
        <p>Jamie's Blog</p>
        <div>
          <a href="https://j4mie.me">Portfolio</a>
        </div>
      </nav>
    </div>
  );
};

export default Header;
