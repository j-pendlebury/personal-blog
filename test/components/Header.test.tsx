import { render } from "@testing-library/react";
import Header from "../../components/Header";

test("Header test", () => {
  const { container } = render(<Header />);
  expect(container).toBeDefined();
  expect(container).toMatchSnapshot();
});
