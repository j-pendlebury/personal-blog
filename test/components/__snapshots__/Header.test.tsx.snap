// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="gradient-border-line"
    />
    <nav>
      <p>
        Jamie's Blog
      </p>
      <div>
        <a
          href="https://j4mie.me"
        >
          Portfolio
        </a>
      </div>
    </nav>
  </div>
</div>
`;
